testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -130,7 +131,10 @@ def __init__(self):
         self.jump_to_frame.positions = 2 ** 3 * self.first_frame.positions
         self.jump_to_frame.frame = 3
 
-        self.dimensions = np.array([80, 80, 80, 60, 60, 90], dtype=np.float32)
+        self.dimensions = np.array([81.1, 82.2, 83.3, 60, 60, 90],
```

Comment:
Just 75 80 85 should be enough?  And you could make these increment by 0.1 over time